versions.lock

```diff
@@ -8,23 +8,23 @@ com.boundary:high-scale-lib:1.0.6 (2 constraints: 9d14d4a0)
-com.fasterxml.jackson.core:jackson-annotations:2.16.1 (26 constraints: e3af7f88)
-com.fasterxml.jackson.core:jackson-core:2.16.1 (26 constraints: 8014c9db)
-com.fasterxml.jackson.core:jackson-databind:2.16.1 (39 constraints: 2aecbde3)
-com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.16.1 (4 constraints: e765844e)
-com.fasterxml.jackson.dataformat:jackson-dataformat-smile:2.16.1 (1 constraints: 811ca2a4)
-com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.16.1 (4 constraints: 23447c2b)
-com.fasterxml.jackson.datatype:jackson-datatype-guava:2.16.1 (4 constraints: 393ff5b1)
-com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.16.1 (3 constraints: 7030675f)
-com.fasterxml.jackson.datatype:jackson-datatype-joda:2.16.1 (2 constraints: 352b50b0)
-com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.16.1 (3 constraints: 7030675f)
-com.fasterxml.jackson.jaxrs:jackson-jaxrs-base:2.16.1 (2 constraints: dd2ecf69)
-com.fasterxml.jackson.jaxrs:jackson-jaxrs-cbor-provider:2.16.1 (1 constraints: 43197fbb)
-com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.16.1 (2 constraints: 271dbf09)
-com.fasterxml.jackson.module:jackson-module-afterburner:2.16.1 (1 constraints: 9d1c9da6)
-com.fasterxml.jackson.module:jackson-module-blackbird:2.16.1 (1 constraints: b50e635e)
-com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.16.1 (2 constraints: dd2ecf69)
-com.fasterxml.jackson.module:jackson-module-parameter-names:2.16.1 (1 constraints: b50e635e)
+com.fasterxml.jackson.core:jackson-annotations:2.15.3 (26 constraints: e8afcd9d)
+com.fasterxml.jackson.core:jackson-core:2.15.3 (26 constraints: 8e145d16)
+com.fasterxml.jackson.core:jackson-databind:2.15.3 (39 constraints: 39ec9d3f)
+com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.15.3 (4 constraints: e8655c4f)
+com.fasterxml.jackson.dataformat:jackson-dataformat-smile:2.15.3 (1 constraints: 811ca2a4)
+com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.15.3 (4 constraints: 24442b2c)
+com.fasterxml.jackson.datatype:jackson-datatype-guava:2.15.3 (4 constraints: 3a3f96b2)
+com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.15.3 (3 constraints: 7130de5f)
+com.fasterxml.jackson.datatype:jackson-datatype-joda:2.15.3 (2 constraints: 352b50b0)
+com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.15.3 (3 constraints: 7130de5f)
+com.fasterxml.jackson.jaxrs:jackson-jaxrs-base:2.15.3 (2 constraints: df2e0e6a)
+com.fasterxml.jackson.jaxrs:jackson-jaxrs-cbor-provider:2.15.3 (1 constraints: 43197fbb)
+com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.15.3 (2 constraints: 271dbf09)
+com.fasterxml.jackson.module:jackson-module-afterburner:2.15.3 (1 constraints: 9d1c9da6)
+com.fasterxml.jackson.module:jackson-module-blackbird:2.15.3 (1 constraints: b50e635e)
+com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.15.3 (2 constraints: df2e0e6a)
+com.fasterxml.jackson.module:jackson-module-parameter-names:2.15.3 (1 constraints: b50e635e)
```

Comment:
I've bumped a few libraries specifically to drop the jackson dependency version -- this currently pushes other projects to a 2.16.1 verison with gradle 7.x incompatibilities